web/src/components/tasks/MultipleDwellingRegistrationTask.tsx

```diff
@@ -0,0 +1,172 @@
+import { Content } from "@/components/Content";
```

Comment:
Given the parallels between this and the other similar 2-tab status check components (License check being one), it might be a cool refactor to have a common component these are both leveraging.

Reply:
I'll create a ticket for this to get tackled on a refactor Friday!